README.md

```diff
@@ -31,9 +35,77 @@ current price of the asset price on the [Coinbase Exchange](https://exchange.coi
 If the sample application is used to liquidate large positions, there is price action risk that may
 result in trades executing up to 10% lower than the latest price check.
 
+## Usage
+
+### Create Stack
+
+The *Coinbase Prime Liquidator* has a [sample AWS CloudFormation](infra/aws.cfn.yml) (CFN) template that can be deployed to run the application in an Amazon Elastic Container Service (Amazon ECS) cluster. This template creates all of the required resources and can be customized to suit the deployers needs. To deploy the CFN stack, [initialize your AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) and then run:
```

Comment:
Should we add a warning here that these resources will/can cost money if deployed and left idle